tests/src/Functional/RequiredElementsTest.php

```diff
@@ -48,10 +48,10 @@ public function testRequiredProperty(): void {
     // Adding an item will trigger the error validation for this multi-value
     // element.
     $assert_session->buttonExists('partial_required_add_more')->press();
-    $this->assertText('Baz field is required.');
+    $this->assertSession()->responseContains('Baz field is required.');
```

Comment:
What is the reason for this change? Is it something that has changed in the way the module works? Keep in mind that `::responseContains()` is checking the page source while `::pageTextContains()` only in visible text. So, I think the later is more appropriate in this case.